docs/_docs/monitoring-metrics/new-metrics.adoc

```diff
@@ -115,6 +115,8 @@ Register name: `cache.{cache_name}.{near}`
 |RollbackTime|histogram|   Rollback time in nanoseconds.
 |RollbackTimeTotal   |long|The total time of rollback, in nanoseconds.
 |TotalRebalancedBytes|long|Number of already rebalanced bytes.
+| PutAllConflictTime | histogram | PutAllConflict time for which this node is the initiator, in nanoseconds.
```

Comment:
1. Excess leading white space
2. Metrics list is sorted in alphabetical order.

Reply:
Fixed it